templates/core/index.html

```diff
@@ -1,817 +1,753 @@
+    <div class="col-6 col-start-large-4 col-medium-4 col-start-medium-3">
+      <p>
+        <a href="/core/docs">Get started with Ubuntu Core&nbsp;&rsaquo;</a>
+      </p>
     </div>
   </div>
-  <div class="u-fixed-width">
-    <p>Check the full list of <a href="/internet-of-things">industries</a> and <a href="/core/stories">success cases&nbsp;&rsaquo;</a></p>
-  </div>
 </section>
 
-<section class="p-strip--light" id="security-first">
-  <div class="u-fixed-width">
-    <h2>Key Features</h2>
-    <h3 class="u-sv2">Security first</h3>
-  </div>
-  <div class="row">
-    <div class="col-4">
-      <div class="p-media-object">
-        <img src="https://assets.ubuntu.com/v1/439b2228-Confinement-small.svg" class="p-media-object__image" alt="">
-        <div class="p-media-object__details">
-          <h3 class="p-media-object__title">
-            <a href="https://snapcraft.io/docs/snap-confinement">Strict confinement </a>
-          </h3>
-          <p class="p-media-object__content p-heading--5" style="margin-bottom: 0;">Keep a bad app in its box</p>
-          <p class="p-media-object__content">You trust your developers, but anybody can make a mistake. We created and integrated a whole new armoury of Linux security capabilities to ensure applications stay confined to their own data.</p>
+<section class="p-section">
+  <div class="u-fixed-width p-section--shallow">
+    <hr class="p-rule" />
+    <h2>For smart use cases. Get Core. Get snaps. Get going.</h2>
+  </div>
+  <div class="row--25-75 p-section--shallow">
+    <div class="col">
+      <hr class="u-hide--small is-muted p-rule" />
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <a href="/automotive">
+            <h3 class="p-heading--5">Automotive</h3>
+            {{ image (
+                url="https://assets.ubuntu.com/v1/f867c883-For-smart-use-cases__Automotivew.png",
+                alt="",
+                width="285",
+                height="160",
+                hi_def=True,
+                loading="auto",
+                ) | safe
+            }}
+          </a>
         </div>
-      </div>
-    </div>
-    <div class="col-4">
-      <div class="p-media-object">
-        <img src="https://assets.ubuntu.com/v1/0091c970-1C-10+year.svg" class="p-media-object__image" alt="">
-        <div class="p-media-object__details">
-          <h3 class="p-media-object__title">
-            10 year security update commitment
-          </h3>
-          <p class="p-media-object__content p-heading--5" style="margin-bottom: 0;">That's a decade of sleeping soundly</p>
-          <p class="p-media-object__content">Ubuntu Core gets 10 years of Canonical maintenance. Your smallest devices are now as secure as your servers. No other embedded Linux comes close.</p>
+        <div class="col-3 col-medium-2">
+          <a href="/industrial">
+            <h3 class="p-heading--5">Industrial</h3>
+            {{ image (
```

Comment:
I've gotten this feedback from Akbar recently so I'm passing it along, please remove the whitespace between `image` and the parenthesis and rerun djlint for all of these to reformat